src/components/NavBar/NavBar.component.jsx

```diff
@@ -26,12 +30,13 @@ import {
 import AvatarImg from '../Avatar';
 
 const NavBar = () => {
+  const history = useHistory();
   const [searchString, setSearchString] = useState('');
 
-  const { toggleDarkMode, isDarkMode } = useDark();
+  const { globalState, dispatchGlobal } = useGlobalContext();
```

Comment:
Here you could use your `globalState` to set the initial value for your `useState` hook.